src/Webhooks.ts

```diff
@@ -57,6 +57,7 @@ type WebhookObject = {
     cryptoProvider: StripeCryptoProvider
   ) => Promise<WebhookEvent>;
   generateTestHeaderString: (opts: WebhookTestHeaderOptions) => string;
+  _stripe: StripeObject | null;
```

Comment:
Did you have to reference the entire `_stripe` object because _platformFunctions were not set when webhooks were constructed?

Reply:
Oh that's a good callout, I didn't need to reference the entire `_stripe` object. I think this would be best resolved if we abstracted the entire getUname function and set that on Webhooks directly, since that's the only place where a platform-specific function is necessary.